src/ServiceControl.AcceptanceTests/ConfigureEndpointMsmqTransport .cs

```diff
@@ -18,6 +18,8 @@ public Task Configure(string endpointName, EndpointConfiguration configuration,
         queueBindings = settingsHolder.Get<QueueBindings>();
 
         var transportConfig = configuration.UseTransport<MsmqTransport>();
+
+        transportConfig.Transactions(TransportTransactionMode.SendsAtomicWithReceive);
```

Comment:
Discussed with @andreasohlund and we agreed that SC doesn't assume DTC transaction, therefore, it is fine to leave this here